src/app/components/pages/TeacherRequest.tsx

```diff
@@ -43,64 +44,42 @@ export const TeacherRequest = () => {
   const errorMessage = useAppSelector((state: AppState) => (state && state.error) || null);
   const { data: warningFragment } = isaacApi.endpoints.getPageFragment.useQuery(warningFragmentId);
 
-  const [firstName, setFirstName] = useState((user?.loggedIn && user.givenName) || "");
-  const [lastName, setLastName] = useState((user?.loggedIn && user.familyName) || "");
-  const [emailAddress, setEmailAddress] = useState((user?.loggedIn && user.email) || "");
+  const emailAddress = (user?.loggedIn && user.email) || "";
```

Comment:
Small one, but it would be tidier to move this one and `emailVerified` next to `urn`, so that you have all the states grouped together, and all the variables grouped together.